src/components/flashcard-set/flashcard-set.tsx

```diff
@@ -66,8 +66,13 @@ export const FlashcardSet = ({
 
   function handleUpClick(index: number) {
     const newCards = [...cards];
```

Comment:
I really like this change. I was thinking the behavior was weird but didn't quite realize why😅

Reply:
perks of writing tests I guess, I was bashing my head for 30 minutes trying to find out why my test didn't work.
lo and behold, it turns out it just didn't do what we expected 😆